### PR TITLE
Feature/logging task 1

### PR DESCRIPTION
**Pull request by serhiiyasenev**

### CODE REVIEW

File: lesson9-Logging/BrainstormSessions/Api/IdeasController.cs [hunk 1]
```diff
@@ -99,15 +101,19 @@ public async Task<ActionResult<List<IdeaDTO>>> ForSessionActionResult(int sessio
         [ProducesResponseType(404)]
         public async Task<ActionResult<BrainstormSession>> CreateActionResult([FromBody]NewIdeaModel model)
         {
+            _logger.Debug($"Start of CreateActionResult Method execution with model {nameof(model)}");
```
Comment (YNauholnykh) [inline lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: So the message will be `Start of CreateActionResult Method execution with model model`?
Cause nameof returns the name of the variable in this case.

File: lesson9-Logging/BrainstormSessions/Api/IdeasController.cs [hunk 2]
```diff
@@ -99,15 +101,19 @@ public async Task<ActionResult<List<IdeaDTO>>> ForSessionActionResult(int sessio
...
+
             if (!ModelState.IsValid)
             {
+                _logger.Error($"ModelState {nameof(model)} is not valid");
```
Comment (YNauholnykh) [inline lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: Same here. Also `ModelState` here represents a state of a model, with some properties and metadata of a model. So I'd not say that the model state is invalid, but model is invalid.

File: lesson9-Logging/BrainstormSessions/Api/IdeasController.cs [hunk 3]
```diff
@@ -121,6 +127,8 @@ public async Task<ActionResult<BrainstormSession>> CreateActionResult([FromBody]
 
             await _sessionRepository.UpdateAsync(session);
 
+            _logger.Debug($"Finish of CreateActionResult Method execution with model {nameof(model)}");
```
Comment (YNauholnykh) [inline lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: Same

File: lesson9-Logging/BrainstormSessions/Controllers/HomeController.cs [hunk 4]
```diff
@@ -42,8 +51,11 @@ public class NewSessionModel
         [HttpPost]
         public async Task<IActionResult> Index(NewSessionModel model)
         {
+            _logger.Debug($"Start of Index Method execution with model {model.SessionName}");
+
             if (!ModelState.IsValid)
             {
+                _logger.Warn($"ModelState is not valid {nameof(model)}");
```
Comment (YNauholnykh) [inline lesson9-Logging/BrainstormSessions/Controllers/HomeController.cs]: Same

File: lesson9-Logging/BrainstormSessions/Api/IdeasController.cs (hunk 1 above)
Comment (serhiiyasenev) [reply lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: yes, we can think about a helper that allows us to print it in `Key: Value` view

Comment (serhiiyasenev) [reply lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: done

File: lesson9-Logging/BrainstormSessions/Api/IdeasController.cs (hunk 2 above)
Comment (serhiiyasenev) [reply lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: done

File: lesson9-Logging/BrainstormSessions/Api/IdeasController.cs (hunk 3 above)
Comment (serhiiyasenev) [reply lesson9-Logging/BrainstormSessions/Api/IdeasController.cs]: done

File: lesson9-Logging/BrainstormSessions/Controllers/HomeController.cs (hunk 4 above)
Comment (serhiiyasenev) [reply lesson9-Logging/BrainstormSessions/Controllers/HomeController.cs]: done

**Review by YNauholnykh — APPROVED** [review]

**Pull request merged**